src/utils/survey.ts

```diff
@@ -0,0 +1,238 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { callWithTelemetryAndErrorHandling, type IActionContext } from '@microsoft/vscode-azext-utils';
+import * as semver from 'semver';
+import { env, Uri, window } from 'vscode';
+import * as nls from 'vscode-nls';
```

Comment:
please use vscode.l10n object

Reply:
Since your localisation PR is not merged yet, I'd rather avoid this change here and then update your PR after merging.